### PR TITLE
Increase maximum chunk size to adjust for small networks.

### PR DESCRIPTION
**Pull request by eskimor**
And warn on invalid merkle proofs in availability recovery.

### CODE REVIEW

**Review by drahnr — APPROVED** [review]
LGTM!

File: node/network/availability-recovery/src/lib.rs [hunk 1]
```diff
@@ -393,7 +393,7 @@ impl RequestChunksPhase {
 						metrics.on_chunk_request_invalid();
 						self.error_count += 1;
 
-						tracing::debug!(
+						tracing::warn!(
```
Comment (rphmeier) [inline node/network/availability-recovery/src/lib.rs:396]: Warning on invalid behavior can be extremely noisy. Nodes misbehaving is abnormal but within expectations, so we typically don't 'warn' in such cases.

Comment (rphmeier) [reply node/network/availability-recovery/src/lib.rs:396]: Still unaddressed

Comment (eskimor) [reply node/network/availability-recovery/src/lib.rs:396]: Yep, alert on `on_chunk_request_invalid` would probably be better.

**Pull request merged**